templates/part.board.mainView.php

```diff
@@ -5,12 +5,12 @@
 		<p>{{ statusservice.text }}</p></div>
 </div>
 
-<div id="board-header">
-	<div id="app-navigation-toggle" class="icon-menu" ng-click="navibar.show=!navibar.show"></div>
-	<a class="crumb" href="#" title="<?php p($l->t('All Boards')); ?>">
-		<i class="icon icon-home"></i>
-		<span class="hidden-visually"><?php p($l->t('All Boards')); ?></span>
-	</a>
+<div id="controls">
```

Comment:
please revert to previous version of this code. It was decided some time ago that it's a good approach. CHanges here are not needed to make toggle work.

I do agree with renaming `#board-header` to `#controls`